mitodl/mitxonline
# Updates the course about page CSS

# What are the relevant tickets?

https://github.com/mitodl/mitxonline/issues/1862

# Description (What does it do?)

This wraps the feature-flagged class changes (and just those changes) in a selector that should only apply if the feature flag is tagged on.

# How can this be tested?

Set the feature flag to Enabled and check - it should look the same as it did in https://github.com/mitodl/mitxonline/pull/1816 . 

Set the feature flag to Disabled and check - it should successfully revert to the old design.
